Support installing without cert-manager

jsok opened the pull request:
Enables users to supply their own certificate for the admission
webhooks.

Closes #531

toast-gear commented:
@jsok `openssl` is probably better given the project implementation is fairly technology agnostic, a link will be fine

jsok commented:
I've fixed some conflicts and added a link to what seems like a trustworthy and also stable set of documentation.
Realistically I think anyone who is going to be doing this already has a mechanism or their own googling skills to work it out 🤷 .
